perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -62,6 +62,7 @@
 MYSQL_SVC_OLTP_TABLES_COUNT = 'mysql_svc_oltp_tables_count'
```

Comment:
I would like to leave them in one PR. I have updated the PR description to reflect both of the new features in this PR.